smartseq2_single_sample/hca-dcp/submit_ss2_single_sample.wdl

```diff
@@ -0,0 +1,37 @@
+task submit {
+  String i
+
+  command {
+    cat ${i} > out.txt
+    echo s >> out.txt
+  }
+  runtime {
+    docker: "python:2.7"
```

Comment:
is this docker image required here? I don't see any python code.

Reply:
Sorry that is a bit confusing. Cromwell requires a docker image for each task. I usually use the python image as a default, just to have an image for the task. However, you have a good point - that can be confusing. I have switched it to ubuntu.